text/chapter1.md

```diff
@@ -133,11 +133,17 @@ This book aims to provide an introduction to the PureScript language for beginne
 
 If you get stuck at any point, there are a number of resources available online for learning PureScript:
 
-- The PureScript IRC channel is a great place to chat about issues you may be having. Point your IRC client at irc.freenode.net, and connect to the #purescript channel.
-- The [PureScript website](https://www.purescript.org) contains links to several learning resources, including code samples, videos and other resources for beginners.
+- The `#purescript` and `#purescript-beginners` FP Slack channels are a great place to chat about issues you may be having. [Use this link](https://fpchat-invite.herokuapp.com/) to gain access the Slack chatrooms.
+- [PureScript: Jordan's Reference](https://github.com/jordanmartinez/purescript-jordans-reference) is an in-depth learning resource. A few things need to be said about this "reference" resource in comparison to this book:
+    - Whereas this book is still being updated, the "reference" repo is up-to-date.
+    - Whereas this book focuses on helping new learners get productive quickly, the "reference" repo helps learners develop very strong conceptual foundations, so that learns can more easily understand advanced concepts later.
+    - Whereas this book has exercises, the "reference" repo does not currently have exercises.
+    - The "reference" repo isn't SEO-friendly. Google searches will not find its content. You will need to use GitHub's search tool to find things in the repo.
+    - If you are not able to understand a concept after reading through this book's content on that topic, consider reading through the "reference" repo's corresponding explanation.
```

Comment:
Can this be consolidated to a single line?

Maybe something like:

> Jordan's Purescript Reference is an alternative learning resource that goes into great depth. If a concept in this book is difficult to understand, consider reading the corresponding section in this reference.

The other points mentioned (such as how to search) seem more appropriate to mention in your reference.

Reply:
That makes sense. Perhaps these comparisons should be written in my repo rather than here in this book? Then, those who click on the link can see that comparison when they view my repo.